octobot_tentacles_manager/api/creator.py

```diff
@@ -76,7 +81,24 @@ async def create_all_tentacles_bundle(output_dir: str = constants.DEFAULT_EXPORT
                 should_zip=should_zip_bundle,
                 should_remove_artifacts_after_use=should_remove_artifacts_after_use)
             await tentacle_bundle_exporter.export()
+            tentacle_bundle_exported_list.append(tentacle_bundle_exporter)
         except Exception as e:
             logger.error(f"Error when exporting tentacle {tentacle.name} : {str(e)}")
             error_count += 1
+
+    if upload_url is not None:
+        for exported_tentacle_bundle in tentacle_bundle_exported_list:
+            await _upload_exported_tentacle_bundle(upload_url, exported_tentacle_bundle)
```

Comment:
should we use 
`await asyncio.gather(*[
    _upload_exported_tentacle_bundle(upload_url, exported_tentacle_bundle)
    for exported_tentacle_bundle in tentacle_bundle_exported_list
])`
to use the uploaded asynchronousness ?

Reply:
Good idea :+1: